### PR TITLE
Partially fix #112: Record email addresses from CFF

### LINKED ISSUE #112: CFF process workflow doesn't record email addresses (and potentially other data)
This should be added in the cff process module, and be accessed through a generic entry method that also calls existing methods.

### PR DESCRIPTION
**Pull request by sdruskat**
This PR is a partial fix for #112, in that it uses the branch for [this PR against cffconvert](https://github.com/citation-file-format/cff-converter-python/pull/294) to record email addresses in CFF files.

## Review checklist

- [x] Run
```bash
git fetch origin
git switch feature/record-cff-email
git pull
poetry shell
poetry install
pytest test/
```
... and all tests should pass.
- [x] Review changed files manually
- [x] Run HERMES against this local repo and check in the `.hermes` files if emails made it over from CFF.

### CODE REVIEW

File: test/hermes_test/commands/harvest/test_cff.py [hunk 1]
```diff
@@ -64,3 +64,44 @@ def test_validate_success(valid_minimal_cff):
 
 def test_validate_fail():
     assert harvest._validate(pathlib.Path("foobar"), {}) is False
+
+
+
```
Comment (led02) [inline test/hermes_test/commands/harvest/test_cff.py]: Too many blank lines...
```suggestion


```

File: test/hermes_test/commands/harvest/test_cff.py [hunk 2]
```diff
@@ -64,3 +64,44 @@ def test_validate_success(valid_minimal_cff):
...
+          "@type": "Person",
+          "email": "em@il.notexist",
+          "name": "Author"
+        }
+      ],
+      "name": "Title"
+    }""")
+
+
+@pytest.fixture()
```
Comment (led02) [inline test/hermes_test/commands/harvest/test_cff.py:87]:
```suggestion
@pytest.fixture
```

... just for consistency ;)

File: test/hermes_test/commands/harvest/test_cff.py (hunk 1 above)
Comment (sdruskat) [reply test/hermes_test/commands/harvest/test_cff.py]: Done in local commit that I forgot to push.

File: test/hermes_test/commands/harvest/test_cff.py (hunk 2 above)
Comment (sdruskat) [reply test/hermes_test/commands/harvest/test_cff.py]: Done in local commit that I forgot to push.

**Review by led02 — APPROVED** [review]
**praise**: Works for me (almost)

**Pull request merged**